### DISCUSSION TITLE
Lab 2 Feedback

### DISCUSSION THREAD

**Discussion by ering0427** [ering0427/Interactive-Lab-Hub#12] [General]
Appreciate any feedback for part G!

**Comment by Rpoddar1953** [comment]
So from the video it looks like you have one button which shows the date and time when pressed and another button which shows the Cornell tech logo. I like the simple interaction you have included with the buttons. It might be interesting to combine the two so that you have the time and date on top of an image like the Cornell tech logo?

**Reply by vbartle** [reply]
Fun idea to center time around Cornell Tech. Adding to Ritika's comment^, it might be fun to overlay time on the logo and let the user switch between orienting time within or above the logo. It might be difficult for the user to read the time, too, so making the font larger or increasing the font size might help legibility.

**Comment by mintae0424** [comment]
Great start! I also agree with both Ritika and Vince. I think it would be better if you could overlay time on the logo. What I thought was interesting is how you have the display off until there is an interaction from the user. I didn't think of it beforehand, but having the display off might actually be better than having the display on continuously.

**Comment by jacobrauch** [comment]
This is pretty cool that you included the images in the background. Another feature that could be added would be to integrate different images depending on the time of day or weather.
